packages/e2e-test/src/commands/run.ts

```diff
@@ -383,8 +383,9 @@ async function testAppServe(pluginName: string, appDir: string) {
     }
   } finally {
     // Kill entire process group, otherwise we'll end up with hanging serve processes
-    await new Promise<void>((res, rej) =>
-      killTree(startApp.pid, err => (err ? rej(err) : res())),
+    await new Promise<void>(
+      (res, rej) =>
+        startApp.pid && killTree(startApp.pid, err => (err ? rej(err) : res())),
```

Comment:
if startApp.pid is falsy, this will never resolve - will it?

Reply:
Yeah just wanted to make tsc pass first, will fix this and the other occurrence too 👍